probe_scraper/scrapers/git_scraper.py

```diff
@@ -33,20 +33,29 @@ def retrieve_files(repo_info, cache_dir):
     try:
         for (ptype, rel_path) in all_files:
             hashes = get_commits(repo, rel_path)
-            for _hash, ts in hashes.iteritems():
+            for _hash, ts in hashes.items():
                 disk_path = os.path.join(base_path, _hash, rel_path)
                 if not os.path.exists(disk_path):
                     contents = get_file_at_hash(repo, _hash, rel_path)
+                    print("Contents of " + disk_path)
+                    print(contents)
+                    print(type(contents))
 
                     dir = os.path.split(disk_path)[0]
+                    print("dir: " + dir)
                     if not os.path.exists(dir):
                         os.makedirs(dir)
                     with open(disk_path, 'wb') as f:
-                        f.write(contents)
-
+                        f.write(contents.encode("UTF-8"))
+                    print("Hash: " + str(_hash))
+                    print("ptype: " + str(ptype))
+                    print("disk path: " + str(disk_path))
+                    
                 results[_hash][ptype].append(disk_path)
+                print(results[_hash][ptype])
                 timestamps[_hash] = ts
     except Exception:
+        print("Got an exception!")
```

Comment:
Are these new prints for keeps?

Reply:
Wow, could have sworn I removed them before committing. Thanks!